### PR TITLE
Update logzio-go (v1.0.0 -> v1.0.1) + Add relevant parameters and tests

### PR DESCRIPTION
**Pull request by yotamloe**

### CODE REVIEW

**Review by Doron-Bargo — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -194,14 +194,25 @@ you can override region settings by using these environment variables.
...
 
 | Parameter | Description |
 |---|---|
 | CUSTOM_QUEUE_DIR| Path to a directory you want to store the queue in, the default value is `$HOME/tmp`|
 
+You can configure Jaeger-Logz.io the save the queue in memory and set log count limit and queue capacity:
+
+| Parameter | Description | Default value |
+|---|---|---|
+| IN_MEMORY_QUEUE| If the parameter is set to `true`, the queue wil be saved in memory| `false` |
```
[the remark this answers is not in the document]
Comment (yotamloe) [reply README.md:209]: @Doron-Bargo Yes if this flag is true, the sender will be initialized with in-memory queue, so `custom_queue_dir` will not be used regardless of its value

File: store/config.go [hunk 2]
```diff
@@ -59,36 +73,93 @@ func ParseConfig(filePath string, logger hclog.Logger) (*LogzioConfig, error) {
 	var logzioConfig *LogzioConfig
 	if filePath != "" {
 		logzioConfig = &LogzioConfig{}
+		// Set default values
+		logzioConfig.LogCountLimit = defaultLogCountLimit
+		logzioConfig.Compress = true
```
Comment (Doron-Bargo) [inline store/config.go:78]: for logzio go ( which is logs mainly ) we need the gzip
Did you check if tracing is working with gzip as well
if yes cool 
If not please update to false

File: store/config.go [hunk 3]
```diff
@@ -59,36 +73,93 @@ func ParseConfig(filePath string, logger hclog.Logger) (*LogzioConfig, error) {
...
+		if err != nil {
+			return nil, err
+		}
 		v.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
 		v.SetDefault(regionParam, "")
 		v.SetDefault(customAPIParam, "")
 		v.SetDefault(customListenerParam, "")
 		v.SetDefault(customQueueDirParam, "")
+		v.SetDefault(inMemoryQueueParam, false)
+		v.SetDefault(CompressParam, true)
```
Comment (Doron-Bargo) [inline store/config.go:102]: the same comment as previous regarding compress

File: store/config_test.go [hunk 4]
```diff
@@ -34,16 +35,28 @@ func TestValidate(tester *testing.T) {
 	config.APIToken = ""
 	assert.NoError(tester, config.validate(logger), "validation failed, one of api token or account token can be empty")
 
-	config.CustomQueueDir = fmt.Sprintf("%s",os.Getenv("HOME"))
+	config.CustomQueueDir = fmt.Sprintf("%s", os.Getenv("HOME"))
```
Comment (Doron-Bargo) [inline store/config_test.go:38]: you update the code to use temp dir , can you update the test as well?

**Review by Doron-Bargo — CHANGES_REQUESTED** [review]
please see my comments

File: store/config.go (hunk 3 above)
Comment (yotamloe) [reply store/config.go:102]: @Doron-Bargo, Yes. tracing is working fine with gzip compression, the traces are sent to logz.io and you can see them in the tracing UI.

**Review by Doron-Bargo — APPROVED** [review]
Amazing

**Pull request merged**